site-docs/src/main/paradox/dynamodb/usage/index.md

```diff
@@ -9,28 +9,40 @@ val clientWithCustom = new DADynamoDBClient(dynamoDbAsyncClient)
 val clientWithDefault = DADynamoDBClient()
 ```
 
-The client exposes two methods:
+The client exposes three methods:
 
 ```scala
-def getAttributeValues(dynamoDbRequest: DynamoDbRequest): F[Map[String, AttributeValue]]
+def getItems[T <: Product, K <: Product](keys: List[K], tableName: String)(implicit returnFormat: DynamoFormat[T], keyFormat: DynamoFormat[K]): F[List[T]]
 
 def updateAttributeValues(dynamoDbRequest: DynamoDbRequest): F[Int]
+
+def putItems[T <: Product](tableName: String, items: List[T])(implicit format: DynamoFormat[T]): F[BatchWriteItemResponse]
 ```
 
-The method takes a dynamoDbRequest Case Class:
+The `updateAttributeValues` method takes a dynamoDbRequest Case Class:
 
 ```scala
   case class DynamoDbRequest(tableName: String,
                              primaryKeyAndItsValue: Map[String, AttributeValue],
                              attributeNamesAndValuesToUpdate: Map[String, Option[AttributeValue]])
 ```
 consisting of:
+
 - tableName - the DynamoDB table you want to make the edits in
 - primaryKeyAndItsValue - the primary key and the value to filter it down
   - you can use the .builder and pass the value to the 'type' method ('.s', '.b', '.n' etc)
   - *Warning* adding a value that isn't already present in the table will cause DynamoDb to add another row so watch out for misspellings
 - attributeNamesAndValuesToUpdate - the key = attribute name you want the value of; the value = new value you want to take place of the current one
   - if you are using the .getAttributeValues method, you can set the value to None
+ 
+The `getItems` method takes a table name and a list of case classes of type `K`
+
+The case class for the primary key should match the primary key of the dynamo table. 
+
+If the table has a primary key of id of type string, the case class would be `PrimaryKey(id: String)`
```

Comment:
Maybe "If, for example, the table has a primary key called "id", of type String,..."